src/App.js

```diff
@@ -1,8 +1,14 @@
 import { Route, Routes } from 'react-router-dom';
+import { useEffect } from 'react';
 import './App.css';
 import { MockAPI } from './components';
+import { useData } from './contexts';
 
 function App() {
+  const { state } = useData();
+  useEffect(() => {
+    console.log(state);
+  }, [state]);
```

Comment:
Can remove the `console.log`'s

Reply:
I put console logs to check the data, in the next I will remove thsese